### PR TITLE
add game artworks

### PR DESCRIPTION
**Pull request by MrCraigen**

### CODE REVIEW

**Comment by MrCraigen** [comment]
There are some logos and backgrounds that are missing in steamgriddb for some games.
I comment on the games that lack a logo / background.

File: gamedb.yaml [hunk 1]
```diff
@@ -262,12 +360,47 @@
...
   status: unsupported
 
+- name: Dimension Jump
+  platform: steam
+  id: 564750
+  poster: https://cdn2.steamgriddb.com/file/sgdb-cdn/grid/29a4715e01c0bc217d96855f193c7be9.png
+  background: https://cdn2.steamgriddb.com/file/sgdb-cdn/hero/9a83eabfb7fa303a2d85dbc6f37483e5.png
+  logo: https://cdn2.steamgriddb.com/file/sgdb-cdn/logo/dea184826614d3f4c608731389ed0c74.png
+
+- name: DiRT 3 Complete Edition
```
Comment (alkazar) [inline gamedb.yaml:390]: There is already an entry for this game

Comment (MrCraigen) [reply gamedb.yaml]: Opps didin't see that :) sorry, fixed.

**Comment by Samsagax** [comment]
Oops. I see a lot of quoting issues with this one that I didn't catch. Maybe it could break something but if it works disregard this message.

**Pull request merged**